src/main/cpp/src/getJsonObjectJni.cpp

```diff
@@ -0,0 +1,37 @@
+/*
```

Comment:
File name for JNI should have all first letter Capitalized.

In addition, we may have other JSON functionalities in the future, beyond `getJsonObject`. Thus, it's better to call `JsonUtilsJni.cpp` and `JsonUtils.java` or similar.

Reply:
Thanks, done.